study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainPediatricsUpdates.java

```diff
@@ -0,0 +1,339 @@
+        if (!cfg.hasPath("pdfs")) {
+            return;
+        }
+        for (Config pdfCfg : pdfDataCfg.getConfigList("pdfs")) {
+            pdfBuilder.insertPdfConfig(handle, pdfCfg);
+        }
+    }
+
+    private void insertStudyGovernance(Handle handle, StudyDto studyDto) {
+        Config governanceCfg = eventDataCfg.getConfig("governance");
+        String shouldCreateGovernedUserExprText = governanceCfg.getString("shouldCreateGovernedUserExpr");
+
+        GovernancePolicy policy = new GovernancePolicy(
+                studyDto.getId(),
+                new Expression(shouldCreateGovernedUserExprText));
+        for (Config aomRuleCfg : governanceCfg.getConfigList("ageOfMajorityRules")) {
+            policy.addAgeOfMajorityRule(new AgeOfMajorityRule(
+                    aomRuleCfg.getString("condition"),
+                    aomRuleCfg.getInt("age"),
+                    ConfigUtil.getIntIfPresent(aomRuleCfg, "prepMonths")));
+        }
+
+        policy = handle.attach(StudyGovernanceDao.class).createPolicy(policy);
+        LOG.info("Created study governance policy with id={}, shouldCreateGovernedUserExprId={}, numAgeOfMajorityRules={}",
+                policy.getId(), policy.getShouldCreateGovernedUserExpr().getId(), policy.getAgeOfMajorityRules().size());
+    }
+
+    private interface SqlHelper extends SqlObject {
+
+
+        @SqlQuery("select event_configuration_id "
+                + " from event_configuration c, event_action ea,  event_action_type eat, activity_instance_creation_action aica, "
+                + " activity_instance_status_type aist,"
+                + " event_trigger et, event_trigger_type tt, activity_status_trigger ast, study_activity sa, study_activity sa2"
+                + " where c.event_trigger_id = et.event_trigger_id and tt.event_trigger_type_id = et.event_trigger_type_id"
+                + " and ea.event_action_id = c.event_action_id and eat.event_action_type_id = ea.event_action_type_id"
+                + " and tt.event_trigger_type_code = 'ACTIVITY_STATUS'"
+                + " and eat.event_action_type_code = 'ACTIVITY_INSTANCE_CREATION'"
+                + " and aica.activity_instance_creation_action_id = ea.event_action_id"
+                + " and sa2.study_activity_id = aica.study_activity_id"
+                + " and ast.activity_status_trigger_id = et.event_trigger_id"
+                + " and aist.activity_instance_status_type_id = ast.activity_instance_status_type_id"
+                + " and sa.study_activity_id = ast.study_activity_id"
+                + " and sa.study_activity_code = :sourceActivityCode"
+                + " and sa2.study_activity_code = :targetActivityCode"
+                + " and aist.activity_instance_status_type_code = 'COMPLETE' "
+                + " and c.umbrella_study_id = :studyId")
+        List<Long> findStudyActivityCreationEventIds(@Bind("studyId") long studyId,
```

Comment:
I have recently added `EventDao.getAllEventConfigurationsByStudyId()`. Do you think that will be useful here, instead of writing another custom query?

Reply:
since I already have one tats tested will use it for now. Next time will use EventDao.getAllEventConfigurationsByStudyId()